### PR TITLE
PFO Characterisation feature updates

### PR DESCRIPTION
**Pull request by brucehoward-physics**
Marking as draft for now -- I want to make the PRs to do a CI-based test first.

### CODE REVIEW

**Comment by brucehoward-physics** [comment]
Tagged @cjbackhouse and @PetrilloAtWork as reviewers, let me know if I should tag someone else 😅

File: sbnanaobj/StandardRecord/SRPfoChar.cxx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// \file    SRPfoChar.cxx
+// \brief   SRPfoChar object for Pfo Characterisation (track vs shower) features (MVA inputs).
+// \author  howard@fnal.gov
+////////////////////////////////////////////////////////////////////////
+
+#include "sbnanaobj/StandardRecord/SRPfoChar.h"
+
+namespace caf
+{
+  SRPfoChar::SRPfoChar()
```
Comment (cjbackhouse) [inline sbnanaobj/StandardRecord/SRPfoChar.cxx:11]: Can we capitalize as `SRPFOChar` please?

**Review by cjbackhouse — APPROVED** [review]

File: sbnanaobj/StandardRecord/SRPfoChar.cxx (hunk 1 above)
Comment (brucehoward-physics) [reply sbnanaobj/StandardRecord/SRPfoChar.cxx]: Done and pushed for the class name/constructor. Should I make the change for the file name as well?

**Comment by brucehoward-physics** [comment]
I noticed some warnings in tests... is this expected in some way?
```
[124/655] Checking class versions for ROOT dictionary CAFMaker
Warning in <TInterpreter::ReadRootmapFile>: class  art::Ptr<sbn::ExtraTriggerInfo> found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: class  art::Wrapper<sbn::ExtraTriggerInfo> found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: class  sbn::ExtraTriggerInfo found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: class  sbn::ExtraTriggerInfo::CryostatInfo found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: enum  sbn::bits::triggerSource found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: enum  sbn::bits::triggerLocation found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: enum  sbn::bits::triggerType found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
[125/655] Checking class versions for ROOT dictionary sbnanaobj_StandardRecord
Warning in <TInterpreter::ReadRootmapFile>: class  art::Ptr<sbn::ExtraTriggerInfo> found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: class  art::Wrapper<sbn::ExtraTriggerInfo> found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: class  sbn::ExtraTriggerInfo found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: class  sbn::ExtraTriggerInfo::CryostatInfo found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: enum  sbn::bits::triggerSource found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: enum  sbn::bits::triggerLocation found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
Warning in <TInterpreter::ReadRootmapFile>: enum  sbn::bits::triggerType found in libsbnobj_Common_Trigger_dict.so  is already in libicaruscode_Decode_DataProducts_dict.so 
[654/655] Install the project...
```

**Comment by PetrilloAtWork** [comment]
Those warnings happened in a transition time with a new `sbnobj` and a old `icaruscode`, where `sbn::ExtraTriggerInfo` object was moved to the former but not yet removed from the latter.
It should not happen if your base is a release, or I think if it's after `v09_58_02`.

**Comment by brucehoward-physics** [comment]
My base is a feature branch of `icaruscode` and likely does not have this update. Thanks!

File: sbnanaobj/StandardRecord/SRPfoChar.cxx (hunk 1 above)
Comment (cjbackhouse) [reply sbnanaobj/StandardRecord/SRPfoChar.cxx]: Yes please

Comment (brucehoward-physics) [reply sbnanaobj/StandardRecord/SRPfoChar.cxx]: Done!

**Pull request merged**